dotnet/samples/GettingStartedWithProcesses/Steps/CRMRecordCreationStep.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Events;
+using Microsoft.SemanticKernel;
+using Models;
+
+namespace Steps;
+
+/// <summary>
+/// Step used in the Processes Samples:
+/// - Step_02_AccountOpening.cs
+/// </summary>
+public class CRMRecordCreationStep : KernelProcessStep
+{
+    public static class Functions
+    {
+        public const string CreateCRMEntry = "CreateCRMEntry";
```

Comment:
nit - `.. CreateCRMEntry = nameof(CreateCRMEntry);` for ease of maintenance.

FYI - "nameof" is evaluated at build-time so has no runtime impact.

(multiple files)